packages/dpkg.rb

```diff
@@ -0,0 +1,38 @@
+require 'package'
+
+class Dpkg < Package
+  description 'dpkg, a medium-level package manager for Debian'
+  homepage 'https://anonscm.debian.org/git/dpkg/'
+  version '1.19.0.5'
+  source_url 'https://salsa.debian.org/dpkg-team/dpkg/-/archive/master/dpkg-master.tar.gz'
+  source_sha256 'a64b257d89d95f2e199fb39c6b37bf9de9e15e2d1e1898060fa9c9af3899330e'
+
+    depends_on 'perl' => :build
+    depends_on 'pkgconfig' => :build
+    depends_on 'automake' => :build
+    depends_on 'make' => :build
+    depends_on 'gcc' => :build
+    depends_on 'linuxheaders' => :build
+    depends_on 'flex' => :build
+    depends_on 'bison' => :build
+    depends_on 'icu4c' => :build
+    depends_on 'gettext' => :build
+    depends_on 'autoconf' => :build
+    depends_on 'libtool' => :build
+
+    def self.build
+	    system "sudo mount -o remount,rw -io exec ~/"
```

Comment:
Why do you need to do this?  What are we installing in the user home directory that needs to be executable?